src/applications/gi/containers/LandingPage.jsx

```diff
@@ -111,6 +118,12 @@ export class LandingPage extends React.Component {
     this.props.eligibilityChange(e);
   };
 
+  validateSearchQuery = searchQuery => {
+    this.setState({
+      searchError: searchQuery === '',
+    });
```

Comment:
Should `LandingPage` keep track of this state or would it be better to be delegated to `KeywordSearch`, where you wouldn't have to be concerned with passing the props anywhere else the component is used?

Also, it seems like the `setState` will get called twice for any changes, and the error state is checked differently between the two calls in `KeywordSearch`:

1. `handleKeyUp`, setting `searchError: !(isVetTecSelected(this.props.filters) || value)`
2. `handleChange`, setting `searchError: searchQuery === ''`.

Not sure if that will be desirable.

Reply:
I know this seems quirky and believe me I tried to encompass the validation logic all in one component. But I couldn't figure out how to do it without breaking functionality or creating another 508 defect.

There were a number of issues that kept coming up where the validation message would persist even after data was inputted in the input field. Or if you stepped back to the `Landing Page` from `Search Results` the validation message error would immediately appear. 

Then there was the user case where if you select `Vet Tec` on the `Landing Page` which shouldn't display a validation error message on `Form Submit` when the Search Field is empty.